src/tiledb/cloud/groups.py

```diff
@@ -92,6 +94,45 @@ def info(uri: str) -> object:
     return groups_client.get_group(group_namespace=namespace, group_name=group_name)
 
 
+def update_info(
+    uri: str,
+    *,
+    description: Optional[str] = None,
+    name: Optional[str] = None,
+    logo: Optional[str] = None,
+    tags: Optional[List[str]] = None,
+) -> None:
+    """
+    Update Group Metadata
+
+    :param uri: URI of the group in the form 'tiledb://<namespace>/<group>'
+    :param description: Group description, defaults to None
+    :param name: Group's name, defaults to None
+    :param logo: Group's logo, defaults to None
+    :param tags: Group tags, defaults to None
+    :return: None
+    """
+    namespace, group_name = utils.split_uri(uri)
+    groups_v1_client = client.build(rest_api.GroupsApi)
+    info = {}
+    for kw, arg in inspect.signature(update_info).parameters.items():
+        if arg.kind != inspect.Parameter.KEYWORD_ONLY:
+            # Skip every non-keyword-only argument
+            continue
+
+        value = locals()[kw]
+        if value is None:
+            # Explicitly update metadata
+            continue
+        info[kw] = value
+
+    info = group_update.GroupUpdate(**info)
+    try:
+        return groups_v1_client.update_group(namespace, group_name, group_update=info)
```

Comment:
LGTM as long as this points to https://github.com/TileDB-Inc/TileDB-Cloud-API-Spec/blob/master/openapi-v1.yaml#L9615

Reply:
yy, it points to that endpoint: https://github.com/TileDB-Inc/TileDB-Cloud-Py/blob/92c6cab79f7b461fad0fd65c5cf6d4540a00367a/src/tiledb/cloud/rest_api/api/groups_api.py#L2515